src/workerd/jsg/modules.h

```diff
@@ -370,12 +458,14 @@ v8::MaybeLocal<v8::Promise> dynamicImportCallback(v8::Local<v8::Context> context
   // explicitly as rejected Promises.
   v8::TryCatch tryCatch(isolate);
   try {
-    auto& lock = jsg::Lock::from(isolate);
-    auto what = kj::Path::parse(kj::str(resource_name)).parent().eval(kj::str(specifier));
+    auto& js = jsg::Lock::from(isolate);
+    auto referrerPath = kj::Path::parse(kj::str(resource_name));
+    auto specifierPath = referrerPath.parent().eval(kj::str(specifier));
     // TODO(soon): If kj::Path::parse fails it is most likely the application's fault and yet
```

Comment:
I assume this needs a PR to capnproto to add the functionality?

Reply:
Possibly. That's an entirely separate PR tho. Not going to worry about it here :-)

Reply:
Yeah, It just looked like it might be a fun one that might go on my radar.